djangorarapi/migrations/0001_initial.py

```diff
@@ -0,0 +1,112 @@
+# Generated by Django 3.1.6 on 2021-02-19 17:39
```

Comment:
Remove this file from the .git tracking file by entering this in your terminal:
 ```git rm --cached djangorarapi/migrations/0001_initial.py```
You can use *tab* completion to fill in the file path to make sure it's right.